### DISCUSSION TITLE
Tutorial to understand capabilities of stellarium

### DISCUSSION THREAD

**Discussion by gagan86nagpal** [Stellarium/stellarium#2104] [General]
Hey Can we enumerate the resources (yt videos, blogs, books) to help people use this masterpiece better ?

**Comment by gzotti** [comment]
The little Stellarium team creates the program and the official User Guide, but does not actively produce youtube videos. There are countless video makers, bloggers and authors who are using or presenting Stellarium, or the sky with simulation provided by Stellarium. A handful have asked for permission or just contacted us to say thank you for the software. We don't require this, but see this as friendly act. However, collecting everything is out of our reach. You could probably make a collection to be displayed on stellarium.org! But how long will this remain up-to-date? @alex-w ?

**Reply by alex-w** [reply]
+1 for creating a collection of books or perfect videos and we really may create a page or section on the website for these data, but I have no free time for this task. Maybe community can help here?
